Packs/CommonWidgets/Scripts/RSSWidget/RSSWidget.py

```diff
@@ -0,0 +1,124 @@
+
+
+def parse_feed_data(feed_response: requests.Response) -> FeedParserDict:
+    """Parses the data from the RSS feed.
+
+    Args:
+        feed_response (requests.Response): The raw data from the RSS feed.
+
+    Returns:
+        FeedParserDict: Parsed RSS feed data.
+    """
+    try:
+        return feedparser.parse(feed_response.text)
+    except Exception as err:
+        raise DemistoException(f"Failed to parse feed.\nError:\n{str(err)}")
+
+
+def collect_entries_data_from_response(parsed_feed_data: FeedParserDict) -> List[Dict[str, Any]]:
+    """Collects relevant data from the parsed RSS feed entries.
+
+    Args:
+        parsed_feed_data (FeedParserDict): Parsed RSS feed data.
+
+    Returns:
+        List[Dict[str, Any]]: The data from the RSS feed relevant for the widget.
+    """
+    entries_data: List[Dict[str, Any]] = []
+    if not parsed_feed_data:
+        raise DemistoException("Could not parse feed data.")
+
+    for entry in reversed(parsed_feed_data.entries):
+        if entry:
+            published = email.utils.parsedate(entry.published)
+            if not published:
+                continue
+
+            published_dt = datetime.fromtimestamp(mktime(published))
+            published_formatted = published_dt.strftime('%B %-d, %Y %-I:%M %p')
+
+            entries_data.append(
+                {
+                    'timestamp': published_formatted,
+                    'link': entry.get('link'),
+                    'title': entry.get('title'),
+                    'summary': entry.get('summary')
+                }
+            )
+
+    return entries_data
+
+
+def create_widget_content(entries_data: List[Dict[str, Any]]) -> str:
+    """Creates the human readable text for the widget.
+
+    Args:
+        entries_data (List[Dict[str, Any]]): The data from the  RSS feed relevant for the widget.
+
+    Returns:
+        str: The widget's content.
+    """
+    content: str = ''
+
+    for entry_data in entries_data:
+        content += f'## [{entry_data["title"]}]({entry_data["link"]})\n'
+        content += f'_{entry_data["timestamp"]}_\n'
+        content += f'#### {entry_data["summary"]}\n'
+        content += '---\n'
+
+    return content
+
+
+def main():
+    args = demisto.args()
+
+    client = Client(
+        server_url=args['url'],
+        use_ssl=not args.get('insecure', False),
+        proxy=args.get('proxy', False),
+    )
+
+    try:
+        rss_raw_data = client.get_feed_data()
+        parsed_feed_data = parse_feed_data(rss_raw_data)
+        entries_data = collect_entries_data_from_response(parsed_feed_data)
+        content = create_widget_content(entries_data)
+    except Exception as e:
+        demisto.error(traceback.format_exc())
+        return_error(str(e))
+
+    demisto.results({
+        'Type': EntryType.NOTE,
+        'ContentsFormat': EntryFormat.MARKDOWN,
+        'Contents': content,
+    })
+
+
+if __name__ in ('__builtin__', 'builtins', '__main__'):
```

Comment:
how did you create the script? it should be in the template of `demisto-sdk init`
```suggestion
if __name__ in ('__builtin__', 'builtins', '__main__'):  # pragma: no cover
```